### PR TITLE
fix(UI): Fix very small frame view on Safari

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This PR fixes very small content display on Safari.

Before:

81148925-7e3f8f00-8f7d-11ea-91e4-1ecb89227304

After:
- Firefox

tfirefox

-Chrome

tchrome

-Edge Chromium

tedge

- Safari (soon)

**Fixes** #8692

## Type of change

- [x] Patch fixing an issue (non-breaking change)

## Target serie

- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

Pre-requisite : Use Safari
- Open Centreon (on any page)
- Main content ought to take place at the bottom of the window.

## Checklist

- [x] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: www/front_src/src/App.tsx [hunk 1]
```diff
@@ -153,12 +185,16 @@ class App extends Component<Props, State> {
...
+const mapDispatchToProps = (dispatch: (any) => void): DispatchProps => {
   return {
     fetchExternalComponents: (): void => {
       dispatch(fetchExternalComponents());
     },
   };
 };
 
-export default hot(connect(null, mapDispatchToProps)(App));
+export default hot(connect(null, mapDispatchToProps)(withStyles(styles)(App)));
```
Comment (bdauria) [inline www/front_src/src/App.tsx:200]: tip: ramda pipe would improve the readability

**Comment by bdauria** [comment]
@Thebarda was the issue present only on safari?

**Comment by Thebarda** [comment]
> @Thebarda was the issue present only on safari?

Yes

**Comment by bdauria** [comment]
> > @Thebarda was the issue present only on safari?
> 
> Yes

Did you have the chance to test it on this browser?

**Comment by Thebarda** [comment]
> > > @Thebarda was the issue present only on safari?
> > 
> > 
> > Yes
> 
> Did you have the chance to test it on this browser?

I can't install a recent version of Safari on Windows but @lpinsivy had tested this PR. See the updated ticket: https://centreon.atlassian.net/browse/MON-5474

File: www/front_src/src/App.tsx [hunk 2]
```diff
@@ -153,12 +186,19 @@ class App extends Component<Props, State> {
...
       dispatch(fetchExternalComponents());
     },
   };
 };
 
-export default hot(connect(null, mapDispatchToProps)(App));
+export default pipe(
+  hot,
+  connect(null, mapDispatchToProps),
+)(withStyles(styles)(App));
```
Comment (bdauria) [inline www/front_src/src/App.tsx:204]:
```suggestion
export default pipe(
  hot,
  connect(null, mapDispatchToProps),
  withStyles(styles),
)(App);
```

**Comment by damianog** [comment]
I have the same issue on a Samsung Digital Signage Display.

**Pull request merged**